src/Symfony/Bundle/FrameworkBundle/Test/WebTestCase.php

```diff
@@ -21,6 +21,15 @@
  */
 abstract class WebTestCase extends KernelTestCase
 {
+    use WebTestAssertions;
+
+    public function tearDown()
```

Comment:
I believe `protected` visibility should suffice here. 

What do you think?

Reply:
Yep, didn't remember that the original method is `protected`, thanks 👍